packages/@lightningjs/ui-components/src/components/Tooltip/Tooltip.js

```diff
@@ -93,7 +94,12 @@ export default class Tooltip extends Base {
           type: Bubble,
           w: this._Background.w,
           h: this._Background.h,
-          radius: this.style.radius,
+          radius: getMaxRoundRadius(
+            this.style.radius,
+            this._Background.w / 2,
+            this._Background.h / 2,
```

Comment:
remove the divide by two for both the width and height, and the height should be `this._Background.h - this.style.pointerHeight` to exclude the pointer that is part of that entire background texture and just focus on the height of the bubble part. You can also remove the w/h setters on 91/92, they're redundant from `_updateBackgroundHeight`